src/rendering/batcher/shared/DefaultBatcher.ts

```diff
@@ -0,0 +1,193 @@
+import { ExtensionType } from '../../../extensions/Extensions';
+import { compileHighShaderGlProgram, compileHighShaderGpuProgram } from '../../high-shader/compileHighShaderToProgram';
+import { colorBit, colorBitGl } from '../../high-shader/shader-bits/colorBit';
+import { generateTextureBatchBit, generateTextureBatchBitGl } from '../../high-shader/shader-bits/generateTextureBatchBit';
+import { roundPixelsBit, roundPixelsBitGl } from '../../high-shader/shader-bits/roundPixelsBit';
+import { getBatchSamplersUniformGroup } from '../../renderers/gl/shader/getBatchSamplersUniformGroup';
+import { Shader } from '../../renderers/shared/shader/Shader';
+import { getMaxTexturesPerBatch } from '../gl/utils/maxRecommendedTextures';
+import { Batcher } from './Batcher';
+import { BatchGeometry } from './BatchGeometry';
+
+import type { BatchableMeshElement, BatchableQuadElement } from './Batcher';
+
+let defaultShader: Shader = null;
+
+/** The default batcher is used to batch quads and meshes. */
```

Comment:
if we are ignoring all the other batcher classes we should add that here as well

Reply:
please clarify - not sure i understand the comment :D